frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -95,7 +97,6 @@ export interface IFieldValuesWidgetProps {
 
```

Comment:
The `onChange` props is typed incorrectly as `(value: string[]) => void` even though we can have any value. Fixing this is opening up a new can of worms that does not fall under the scope of this PR.

This does mean we need to ignore some ts errors in the implementation of the component however.